common/common.h

```diff
@@ -32,24 +32,6 @@ template <class E> using UnorderedSet = absl::flat_hash_set<E>;
 // Uncomment to make vectors debuggable
 // template <class T, size_t N> using InlinedVector = std::vector<T>;
 
-#ifdef DEBUG_MODE
```

Comment:
This `if` was inverted, that's the main part of fix.
Also, arguably, the fact that this is build in debug/fuzz/emscripten -mode is version-y, so I moved it there.